src/get-started/java-binding.md

```diff
@@ -222,6 +222,26 @@ reference. Light clients also provide access to information on the
 [transaction][exonum-transaction] execution result
 (which may be either success or failure) to their users.
 
+### Blockchain Events
+
+A service can also handle a block commit event that occurs each time
+the framework commits a new block. The framework delivers this event to
+implementations of [`Service#afterCommit(BlockCommittedEvent)`][service-after-commit]
```

Comment:
such as "callback" word is used below:
"implementations of the callback method..."